Add manuscript links to readme (#1048)
greenelab/covid19-review: README.md

[![Gitter](https://badges.gitter.im/covid19-review/community.svg)](https://gitter.im/covid19-review/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
<!-- usage note: delete CI badges above for services not used by your manuscript -->

Project Status:
| Section | Title | Status Issue | Submission Status | Venue | Links
| -- | -- | -- | -- |-- | -- |

> | [Pathogenesis](https://greenelab.github.io/covid19-review/#pathogenesis-symptomatology-and-transmission-of-sars-cov-2-through-analysis-of-viral-genomics-and-structure) | Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through analysis of Viral Genomics and Structure | #744 | Ready for resubmission | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pathogenesis-v2); [Preprint](http://arxiv.org/abs/2102.01521)

Reviewer: ```suggestion
| [Pathogenesis](https://greenelab.github.io/covid19-review/#pathogenesis-symptomatology-and-transmission-of-sars-cov-2-through-analysis-of-viral-genomics-and-structure) | Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through analysis of Viral Genomics and Structure | ✔️ | Accepted | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pathogenesis-v2); [Preprint](http://arxiv.org/abs/2102.01521)
```

> | Methods (Cyberinfrastructure) | An Open-Publishing Response to the COVID-19 Infodemic | ✔️ | Accepted | [DISCO 2021](https://infoqualitylab.org/events/disco2021/) | [Release](https://github.com/greenelab/covid19-review/releases/tag/methods-v2); [Preprint](https://arxiv.org/abs/2109.08633)

Reviewer: ```suggestion
| [Methods (Cyberinfrastructure)](https://greenelab.github.io/covid19-review/#an-open-publishing-response-to-the-covid-19-infodemic) | An Open-Publishing Response to the COVID-19 Infodemic | ✔️ | Accepted | [DISCO 2021](https://infoqualitylab.org/events/disco2021/) | [Release](https://github.com/greenelab/covid19-review/releases/tag/methods-v3); [Preprint](https://arxiv.org/abs/2109.08633)
```

Updating with the latest release I just made.

| Pharmaceuticals | Identification and Development of Therapeutics for COVID-19 | ✔️ | Accepted | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pharmaceuticals-v3); [Preprint](http://arxiv.org/abs/2103.02723)

> | Pathogenesis | Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through analysis of Viral Genomics and Structure | #744 | Ready for resubmission | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pathogenesis-v2); [Preprint](http://arxiv.org/abs/2102.01521)

Reviewer: ```suggestion
| [Pathogenesis](https://greenelab.github.io/covid19-review/#pathogenesis-symptomatology-and-transmission-of-sars-cov-2-through-analysis-of-viral-genomics-and-structure) | Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through analysis of Viral Genomics and Structure | #744 | Ready for resubmission | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pathogenesis-v2); [Preprint](http://arxiv.org/abs/2102.01521)
```
What do you think about linking the first column to where the paper appears in the merged review?  That will help readers find the absolute latest version if they're interested.  Those links should be stable as long as the section titles do not change.

We can get the URLs from the table of contents in the HTML manuscript.
